assets/js/translator.js

```diff
@@ -1242,6 +1265,43 @@ function autoSelectDstLang() {
     $('#dstLangSelect').val(curDstLang).change();
 }
 
+// Check if the Language pair Exist or Not
+function checkLangPairAvailable(lang) {
+    if(languages[lang] !== undefined) {
+        curSrcLang = iso639CodesInverse[lang];
+        autoSelectDstLang();
+        if(curDstLang !== undefined) {
```

Comment:
This is all equivalent to `return curDstLang !== undefined`

Reply:
Aahh, yup thats right. :)

Reply:
Done. :)